src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -23,7 +23,10 @@
  */
 class FormExtension extends \Twig_Extension implements \Twig_Extension_InitRuntimeInterface
 {
-    private $renderer;
+    /**
+     * Make this property private in 4.0.
+     */
+    public $renderer;
```

Comment:
Maybe a better way might be to keep it private and use a magic getter to throw a deprecation notice?